brain/browser_use_adapter.py

```
@@ -576,14 +663,52 @@ async def close_session(self, session_id: str) -> None:
+        import signal
+        try:
+            import psutil
+            parent = psutil.Process(browser_pid)
+            for child in parent.children(recursive=True):
+                try:
+                    child.kill()
+                except (psutil.NoSuchProcess, psutil.AccessDenied):
+                    pass
+            parent.kill()
+            logger.info("[BrowserUse] force-killed browser tree (pid=%d) via psutil", browser_pid)
+        except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)